devops/ci-python-package.yml

```diff
@@ -0,0 +1,49 @@
+# TODO: Will add trigger part. I omit it for debugging.
+# TODO: Will separate QA, CI and CD to separated files.
+# TODO: Will use variable instead of fixed file name (pyot-0.0.1-py3-none-any.whl)
+
+pool:
+    vmImage: 'ubuntu-latest'
+
+steps:
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: '3.6'
+    addToPath: true
+    architecture: 'x64'
+
+- script: pip install --upgrade pip && pip install -r requirements.txt
+  displayName: 'Install requirements'
+
+- script: pytest 
+  displayName: 'Run test'
+
+- script: python setup.py sdist bdist_wheel
+  displayName: 'Build python package'
+
+- task: PublishBuildArtifacts@1
+  inputs:
+    PathtoPublish: './dist'
+    ArtifactName: 'pyot'
+  displayName: 'Publish python package'
+
+- task: AzureKeyVault@1
+  inputs:
+    azureSubscription: 'masota-cseazure(6e3d5121-9b76-4db1-9f55-877904ca127e)'
```

Comment:
You probably don't want your Azure Sub ID in this commit.

Reply:
Yes, I don't want to commit it. Thank you for finding it. I removed it from file.
After I merge this PR, I'll delete all history for this file. During our repo has the PR, I have following error when I try to delete history.
https://stackoverflow.com/questions/34265266/remote-rejected-errors-after-mirroring-a-git-repository